### PR TITLE
Remove HTMLAttrxRef macro, part 2

### PR DESCRIPTION
**Pull request by OnkarRuikar**
- Continuing https://github.com/mdn/content/pull/25213

The PR starts converting two param instances to `HTML/Element` links.

### CODE REVIEW

File: files/en-us/mozilla/firefox/releases/33/index.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ For details please [see the hacks post](https://hacks.mozilla.org/2014/07/event-
 - For {{HTMLElement("canvas")}}, the method {{domxref("CanvasPattern.setTransform()")}}, allowing to modify a pattern using the {{domxref("DOMMatrix")}} representation of a linear transform ([Firefox bug 1019257](https://bugzil.la/1019257)).
 - Our experimental implementation of Media Source Extensions, behind the `media.mediasource.enabled` preference, enabled by default in Nightly and Aurora only, now supports MP4 ([Firefox bug 1000686](https://bugzil.la/1000686)).
 - The properties {{domxref("HTMLMediaElement.audioTracks")}} and {{domxref("HTMLMediaElement.videoTracks")}} have been experimentally implemented. They are controlled by the `media.track.enabled`, off by default ([Firefox bug 744896](https://bugzil.la/744896)).
-- The non-standard `XMLHttpRequest.mozBackgroundRequest()` is no more accessible from Web sites. Only Firefox-internal code (Chrome code) can use it ([Firefox bug 1035242](https://bugzil.la/1035242)).
+- The non-standard `XMLHttpRequest.mozBackgroundRequest()` is no more accessible from Websites. Only Firefox-internal code (Chrome code) can use it ([Firefox bug 1035242](https://bugzil.la/1035242)).
```
Comment (teoli2003) [inline files/en-us/mozilla/firefox/releases/33/index.md:70]:
```suggestion
- The non-standard `XMLHttpRequest.mozBackgroundRequest()` is no more accessible from websites. Only Firefox-internal code (Chrome code) can use it ([Firefox bug 1035242](https://bugzil.la/1035242)).
```

File: files/en-us/mozilla/firefox/releases/38/index.md [hunk 2]
```diff
@@ -106,9 +106,9 @@ _No change._
...
-- Web sites needing to fall back to an insecure version of TLS in order to work are now in a hardcoded whitelist which will shrink over time ([Firefox bug 1114816](https://bugzil.la/1114816)). The whitelist can be disabled by setting `security.tls.insecure_fallback_hosts.use_static_list` to `false`.
+- In Firefox, the [`autocomplete`](/en-US/docs/Web/HTML/Element/input#autocomplete)`=false` attribute is now ignored when dealing with a login form ([Firefox bug 1025703](https://bugzil.la/1025703)). This is intended to encourage the use of more secure passwords by allowing password manager tools to work more reliably.
+- RC4 is now disabled when using TLS, except for a few specifically whitelisted Websites. This whitelist is an interim measure until those sites are fixed ([Firefox bug 1124039](https://bugzil.la/1124039)). This fallback is controlled by the `security.tls.unrestricted_rc4_fallback` preference, `true` by default for the moment ([Firefox bug 1138882](https://bugzil.la/1138882)).
```
Comment (teoli2003) [inline files/en-us/mozilla/firefox/releases/38/index.md:110]:
```suggestion
- RC4 is now disabled when using TLS, except for a few specifically allowed websites. This whitelist is an interim measure until those sites are fixed ([Firefox bug 1124039](https://bugzil.la/1124039)). This fallback is controlled by the `security.tls.unrestricted_rc4_fallback` preference, `true` by default for the moment ([Firefox bug 1138882](https://bugzil.la/1138882)).
```

File: files/en-us/mozilla/firefox/releases/48/index.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ slug: Mozilla/Firefox/Releases/48
   - The default style for these elements has been adapted to match the spec ([Firefox bug 1258657](https://bugzil.la/1258657)).
   - The {{domxref("HTMLDetailsElement/toggle_event", "toggle")}} event is now sent to the {{HTMLElement("details")}} element when this one is opened or closed ([Firefox bug 1225412](https://bugzil.la/1225412)).
 
-- The {{htmlattrxref("meta", "referrer")}} attributes now also supports the `no-referrer-when-downgrade` and `origin-when-cross-origin` values ([Firefox bug 1178337](https://bugzil.la/1178337)).
+- The [`meta`](/en-US/docs/Web/HTML/Element/referrer#meta) attributes now also supports the `no-referrer-when-downgrade` and `origin-when-cross-origin` values ([Firefox bug 1178337](https://bugzil.la/1178337)).
```
Comment (OnkarRuikar) [inline files/en-us/mozilla/firefox/releases/48/index.md:30]:
```suggestion
- The [`meta`](/en-US/docs/Web/HTML/Element/meta) attributes now also supports the `no-referrer-when-downgrade` and `origin-when-cross-origin` values ([Firefox bug 1178337](https://bugzil.la/1178337)).
```

File: files/en-us/mozilla/firefox/releases/42/index.md [hunk 4]
```diff
@@ -96,7 +96,7 @@ Our experimental implementation of the [Shadow DOM](/en-US/docs/Web/Web_Componen
 
 - The {{domxref('IDBCursorWithValue')}} interface is now available in [Web workers](/en-US/docs/Web/API/Web_Workers_API) ([Firefox bug 1188115](https://bugzil.la/1188115)).
 - Error events sent in [Web workers](/en-US/docs/Web/API/Web_Workers_API) doesn't bubble anymore ([Firefox bug 1188141](https://bugzil.la/1188141)).
-- [Media Source Extensions](/en-US/docs/Web/API/Media_Source_Extensions_API) (MSE) has been activated for all Web sites, and no more for a white list of sites ([Firefox bug 1185611](https://bugzil.la/1185611)).
+- [Media Source Extensions](/en-US/docs/Web/API/Media_Source_Extensions_API) (MSE) has been activated for all Websites, and no more for a white list of sites ([Firefox bug 1185611](https://bugzil.la/1185611)).
```
Comment (teoli2003) [inline files/en-us/mozilla/firefox/releases/42/index.md:99]:
```suggestion
- [Media Source Extensions](/en-US/docs/Web/API/Media_Source_Extensions_API) (MSE) has been activated for all websites, and no more for a white list of sites ([Firefox bug 1185611](https://bugzil.la/1185611)).
```

File: files/en-us/mozilla/firefox/releases/48/index.md [hunk 5]
```diff
@@ -84,7 +84,7 @@ slug: Mozilla/Firefox/Releases/48
 #### DOM & HTML DOM
 
 - Dropped the "Moz" prefix from the {{domxref("CSSKeyframeRule")}} and {{domxref("CSSKeyframesRule")}} interfaces ([Firefox bug 1256178](https://bugzil.la/1256178)).
-- The {{domxref("NavigatorConcurrentHardware")}} {{Glossary("mixin")}} has been implemented, which adds the {{domxref("NavigatorConcurrentHardware.hardwareConcurrency", "window.navigator.hardwareConcurrency")}} property to the {{domxref("Navigator")}} interface. This lets Web sites and apps get at least an approximation of how many processing cores are available to run {{domxref("Worker")}}s in ([Firefox bug 1008453](https://bugzil.la/1008453)).
+- The {{domxref("NavigatorConcurrentHardware")}} {{Glossary("mixin")}} has been implemented, which adds the {{domxref("NavigatorConcurrentHardware.hardwareConcurrency", "window.navigator.hardwareConcurrency")}} property to the {{domxref("Navigator")}} interface. This lets Websites and apps get at least an approximation of how many processing cores are available to run {{domxref("Worker")}}s in ([Firefox bug 1008453](https://bugzil.la/1008453)).
```
Comment (teoli2003) [inline files/en-us/mozilla/firefox/releases/48/index.md:87]:
```suggestion
- The {{domxref("NavigatorConcurrentHardware")}} {{Glossary("mixin")}} has been implemented, which adds the {{domxref("NavigatorConcurrentHardware.hardwareConcurrency", "window.navigator.hardwareConcurrency")}} property to the {{domxref("Navigator")}} interface. This lets websites and apps get at least an approximation of how many processing cores are available to run {{domxref("Worker")}}s in ([Firefox bug 1008453](https://bugzil.la/1008453)).
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
A few nits.

**Review by teoli2003 — APPROVED** [review]

**Pull request merged**